.gitignore

```diff
@@ -27,4 +27,5 @@ hs_err_pid*
 ProductsFromMarketPlace.iml
 
 /.idea/
-/target/
\ No newline at end of file
+/target/
+/Plan.txt
```

Comment:
try settings the .iml files in the gitignore and try to delete them from the repository